leapp/exceptions.py

```diff
@@ -105,3 +105,13 @@ def __init__(self, message):
 class LeappRuntimeError(LeappError):
     def __init__(self, message):
         super(LeappRuntimeError, self).__init__(message)
+
+
+class StopActorExecution(Exception):
+    def __init__(self):
+        super(StopActorExecution, self).__init__()
+
+
+class StopActorExecutionError(LeappError):
+    def __init__(self, message):
+        super(StopActorExecutionError, self).__init__(message)
```

Comment:
You would want also the additional options covered that are covered by report_error such as details and severity

Reply:
Thinking that I am missing function to stop execution. Instead of raw raise of exception, there would be function for that like `stop_execution`, `exit_error`, `exit`, ... Maybe make it part of API.

Reply:
Irrelevant to this, this is a convenience function that would be added on top of it.